api/banners/views.py

```diff
@@ -21,7 +21,6 @@ class CurrentBanner(JSONAPIBaseView, generics.RetrieveAPIView):
 
     serializer_class = BannerSerializer
     # This view goes under the _/ namespace
-    versioning_class = None
```

Comment:
Should this be `PrivateVersioning` instead? Or is the idea we still want to version the endpoint because the field change might break existing banner uses? (The versioning_class for private endpoints is new to me, so some of the details are fuzzy)

Reply:
ahh, I'd never heard of `PrivateVersioning`, I'll look into this, thanks!

Reply:
ok, I switched to `PrivateVersioning` class - for our purposes, I don't think it matters if we use BaseVersioning or PrivateVersioning, more info below.

The reason `versioning_class` could no longer be None is that the `BannerSerializer` had a `VersionedDateTimeField` on it.   This PR modifies the `VersionedDateTimeField`  so a `version` is required on the request.  Both the default versioning and PrivateVersioning both allow `version` on the request.  PrivateVersioning just overrides `reverse` to potentially ignore the `version` arg, if that would cause something to break.  Which isn't the case here.

Perhaps the serializer field shouldn't be VersionedDateTimeField in the first place, just a regular DateTimeField.  I don't have the context as to why this was used in the first place.

Reply:
It probably just got caught in a general update for that field. I think it's safe just to use the new field format or go ahead like this.